src/utils/domain.ts

```diff
@@ -61,11 +71,11 @@ export const formatTick = (axis: IAxis) => (v: string | number) => {
 
 
 interface ITickProps {
-  axis: Axis<string> | Axis<number> | Axis<number | { valueOf(): number}>;
-  axisConfig: IAxes;
+  axis: Axis<string> | Axis<number> | Axis<number | { valueOf(): number }> | Axis<number | string>;
```

Comment:
I might be wrong, but this might be possible to simplify as `Axis<string | number | { valueOf(): number }>`

Reply:
nice try - but unfortunately it doesn't work - ts errors for some of the axis being passed into the function